pkg/helm/export.go

```diff
@@ -41,13 +41,24 @@ func ExportHelmChart(rootFs, fs billy.Filesystem, helmChartPath string, packageV
 	if err != nil {
 		return fmt.Errorf("Cannot parse original chart version %s as valid semver", chart.Metadata.Version)
 	}
-	// Add packageVersion as string, preventing errors due to leading 0s
-	if uint64(packageVersion) >= MaxPatchNum {
-		return fmt.Errorf("Maximum number of packageVersions is %d, found %d", MaxPatchNum, packageVersion)
+	if packageVersion > 0 {
+		// Add packageVersion as string, preventing errors due to leading 0s
+		if uint64(packageVersion) >= MaxPatchNum {
+			return fmt.Errorf("Maximum number of packageVersions is %d, found %d", MaxPatchNum, packageVersion)
+		}
+		chartVersionSemver.Patch = PatchNumMultiplier*chartVersionSemver.Patch + uint64(packageVersion)
+	} else if version != "" {
+		chartVersionSemver, err = semver.Parse(version)
+		if err != nil {
+			return fmt.Errorf("Cannot parse version %s as an valid semver: %s", version, err)
+		}
```

Comment:
We shouldn't even try parsing `semver.Make(chart.Metadata.Version)` if the version is specified. Can you move that call into the packageVersion check?

We should probably restructure this as:

1. If version, take it.
2. Else, parse the chart.Metadata.Version. If packageVersion, append it.